docs/rest_client/models.rst

```diff
@@ -20,10 +20,12 @@ A model needs just enough information to be able to communicate with the API.
 
    import { Model } from 'ponyjs/models.js';
 
-   // provide the model name + declaration of fields/meta/managers
+   // Provide the model name + declaration of fields/meta/managers
+   // If `model_name` is omitted, lower case representation of the model's name is used
```

Comment:
I tried, but it seemed a bit too long and was wrapping on several lines, so it didn't look that good to me.